### ISSUE TITLE
Font Awesome Pro integration

### ISSUE THREAD

**Issue by hmt3design** [soflyy/oxygen-bugs-and-features#1441]
The Font Awesome plugin for Wordpress allows users to enter in an API token to get access to the Pro collection. It would be great if that functionality could be implemented into Oxygen, so that users who have Pro access could access the entire library, not just the Free ones already incorporated into Oxygen.

https://fontawesome.com/how-to-use/on-the-web/using-with/wordpress

**Comment by BrynWithin** [comment]
You can already add all the Font Awesome Pro icons to Oxygen. Just export all the SVGs and then follow the instructions in the Oxygen documentation for creating an icon set file using the free IcoMoon web app.

That said, an official integration would be nice if only so that the duotone icons can be configured effortlessly within Oxygen.

**Comment by octapress** [comment]
Agree! would be a game changer to enter kit details or pro details to get access to all icons.
Or at least a simple way to install fontawesome sub-sets or kits for self hosting.

the icomoon version works, but it's difficult to install the icons in menus etc, they only work well when added as icon boxes etc.

**Comment by stphnwlkr** [comment]
Please add this...the inability to use the pro features in an integrated fashion eliminates Oxygen from my list of production tools on several projects. This will be extremely important with the release of version 6.

**Comment by ayhanmalkoc** [comment]
+1

**Comment by controlauk** [comment]
please add this its a pain to have to create this, also to use font awesome as a font set so can be used in css :before ::after

**Comment by dmg30** [comment]
This would be a great improvement. Love Font Awesome but hate the way that I have to use it with Oxygen.

**Comment by VinceBalk** [comment]
full support for FA 6 would be great.
